src/ColumnSelectionStep/ColumnSelectionStep.tsx

```diff
@@ -164,6 +194,8 @@ export const ColumnSelectionStep: FunctionComponent<ColumnSelectionStepProps> =
 
   return (
     <>
+      {NullsInAidWarning(schema, aidColumn)}
```

Comment:
This should be `<NullsInAidWarning schema={schema} aidColumn={aidColumn} />` because it uses hooks and therefore must be treated like a component. Hook execution must always be deterministic.

Reply:
I'll change this, but I'm not following the explanation or note on determinism. This is some subtle difference somewhere in this JSX thing, right?

Reply:
Hooks are identified by call order, meaning they should never be put in conditionals. In this case you trigger a hook while rendering. That's fine but it's hidden and future refactors can't know that there's a hook call inside the markup.